content/posts/tekton-jx-pipelines.md

```diff
@@ -0,0 +1,954 @@
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-hk95j-1   24m
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-wm85w-6   10h
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-wpqsp-1   10h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw-3   4h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-1         10h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh-2   27m
+
+NAME                                                                AGE
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-d495w   23m
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-hk95j   24m
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-wm85w   10h
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-wpqsp   10h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin         4d
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw   4h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin-rhg88   3d
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jx-mas         10h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh   27m
+```
+
+## Some thoughts and final conclusions
+
+[Traditional Jenkins](https://jenkins.io/) has long been one of the best CI/CD pipeline engines in terms of flexibility, standardization and adoption for most DevOps environments. But new challenges for software delivery requires scalable CI/CD architectures. Today that means using Kubernetes as a powerful abstraction of the infrastructure and the use of cloud native platforms to decouple and scale CI/CD pipelines.
+
+Decoupling and scaling pipelines is one of the best approaches for building new modern applications and microservices. But it can be very hard to work on these decoupled objects to define a pipeline, which conceptually is a sequential and "monolithic" tasks execution.
+
+So, Tekton is demonstrating its power of decoupling CI/CD pipelines and builds to scale. But it also needs a powerful orchestrator to simplify the complexity underneath, and even more when dealing with a platform like Kubernetes, that can also add more complexity.
+
+I like to say that the best way to build and run CI/CD pipelines for modern (and also traditional, why not!) software applications is to make it simple and "abstract the abstraction". I believe Jenkins X is about that. It is about applying the simplicity of trational Jenkins pipelines from a powerful and scalable engine.
+
+Most development companies are realizing that CI/CD next iteration is already here and that it's not only about cloud native pipelines, YAML object definintions or Kubernetes containers orchestration. It is about orchestrating all the complexity while adopting best practices. We are talking about build packs templating, simple YAML pipelines, seamless Git integration, scalable team management, flexible and standard deployments, extensible platform engine, standard packaging and promotion, etc.
```

Comment:
> realizing that CI/CD next iteration is already here

I think this should be reworded to:

realizing that the next iteration of CI/CD is already here